.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,9 +12,11 @@ Checklist for new projects:
 
 - [ ] The project is [open-source](https://opensource.org/licenses/alphabetical) and accessible.
 - [ ] Searched the existing entries to make sure this is not a duplicate.
-- [ ] Added the link to the bottom of the relevant category.
 - [ ] Contains only a single addition (make separate PRs if adding more than one).
-- [ ] Used [title-casing](http://titlecapitalization.com) (AP style) for project name.
+- [ ] Added the link to the bottom of the relevant category.
+- [ ] Add icon from [here](media/icon/) if applicable (like ![Python](media/icon/python.svg)).
```

Comment:
```suggestion
- [ ] Add icon from the `media/icon/` folder if applicable (like `![Python](media/icon/python.png)` ).
```

Reply:
Good point. It's also more readable in "non commit" mode.